docker-compose.yaml

```diff
@@ -0,0 +1,48 @@
+version: "3.3"
+services:
+  decluttarr:
+    image: ghcr.io/manimatter/decluttarr:latest
+    container_name: decluttarr
+    restart: always
+    environment:
+      - TIME_ZONE=Europe/Zurich
```

Comment:
I would stick with "TZ"  since that is - to my knowledge - a docker standard. However, I am not sure whether setting TZ actually chagned the timezone of the decluttarr image.

On a quick google search, many articles talk about `tzdata` package that needs to be installed for ubuntu-based images (which decluttarr is, as you will see in the workflow-yml files). 

Here's smth I found: https://dev.to/bitecode/set-timezone-in-your-docker-image-d22#comment-1h7e2

Do you want to have a look? I think that's relevant since when printing out the timestamps I guess we want the time to be correct ;)

Reply:
I did have a look at the link you mentioned.

The overall consensus was to first install the `tzdata` package and then add the following to your docker file:

```yaml
FROM ubuntu:22.04

ARG DEBIAN_FRONTEND=noninteractive

ENV TZ Europe/Berlin

RUN apt-get update \
&& apt-get install -yq tzdata locales \
&& rm -rf /var/lib/apt/lists/* \

&& sed -i '/en_US.UTF-8/s/^# //g' /etc/locale.gen \
&& locale-gen \
&& ln -fs /usr/share/zoneinfo/${TZ} /etc/localtime \
&& dpkg-reconfigure tzdata

ENV LANG en_US.UTF-8

ENV LANGUAGE en_US:en
ENV LC_ALL en_US.UTF-8

CMD ["/bin/bash"]
```

At the minimum, this might work:

```
FROM alpine:latest

RUN apk update && apk add tzdata
ENV TZ="America/New_York"
```

It is interesting to note this is not an issue on debian based images.

Reply:
I don't really see a reason why we couldn't change to from ubuntu to alpine. 

> At the minimum, this might work:
> 
> ```
> FROM alpine:latest
> 
> RUN apk update && apk add tzdata
> ENV TZ="America/New_York"
> ```

If this gives us correct system time stettings, I think we should go for it

Reply:
> I don't really see a reason why we couldn't change to from ubuntu to alpine

I don't know enough about docker to comment on the difference between the two. But I did find out that `python:3.10.13-slim` is actually not ubuntu but debian...

The change is not neccessary as my tests revealed that `tzdata` is already a package installed in `python:3.10.13-slim`. All I did was upgrade it. The only neccessary changes are what I mentioned in my comment `Proposed Solution` below